src/adapters/poi/idunn_poi.js

```diff
@@ -81,28 +81,35 @@ export default class IdunnPoi extends Poi {
     }
   }
 
-  static async poiApiLoad(id, options = {}) {
+  static async poiApiLoad(obj, options = {}) {
     let rawPoi = null;
-    const url = `${serviceConfig.idunn.url}/v1/places/${id}`;
+    const url = `${serviceConfig.idunn.url}/v1/places/${obj.id}`;
     let requestParams = {};
     if (options.simple) {
       requestParams = {verbosity: 'short'};
     }
     try {
-      rawPoi = await Ajax.getLang(url, requestParams);
+      let headers = {};
+      if (obj.suggestionRank !== undefined && obj.query !== undefined) {
+        if (obj.position !== undefined) {
+          const pos = obj.position;
```

Comment:
Why do you need to copy it? If it's just to have a shorter line below, a better solution would be to deconstruct the object:
```js
const { lat, lon, zoom } = obj.position;
```

Reply:
:+1: